metro/proto/v1/spec.proto

```diff
@@ -123,6 +123,28 @@ service AdminService {
         };
     }
 
+    rpc FetchProjectCredentials(ProjectCredentials) returns (ProjectCredentials){
+        option (google.api.http) = {
+            get: "/v1/projects/{project_id}/credentials/{username}"
+        };
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Fetch Project Credentials for the given username"
+            description: "Fetch the project credentials from username"
+            tags: "Project"
+        };
+    }
+
+    rpc ListProjectCredentials(ProjectCredentials) returns (ProjectCredentialsList){
+        option (google.api.http) = {
+            get: "/v1/projects/{project_id}/credentials/list"
```

Comment:
Do we actually need to add the verb `list` to the API? It's a GET request. Hence feel like we can keep the API as GET `/v1/projects/{project_id}/credentials`?